scripts/lib/versions.js

```diff
@@ -86,17 +86,24 @@ async function getAnnotations(){
   const output = await run(cmd);
   const lines = output.split('\n').filter(Boolean);
   return lines.map(line => {
-    const r = /(.*(vaadin-.*)-parent.*):(.*value *= *"([^"]+).*version *= *"((\d+)\.(\d+)[^"]*).*)/.exec(line);
-    return {
-      path: r[1],
-      name: r[2],
-      annotation: r[3],
-      package: r[4],
-      version: r[5],
-      major: r[6],
-      minor: r[7],
-      updatedVersion: ''
-    };
+    try {
+      const r = /(.*(vaadin-.*)-parent.*):(.*value *= *"([^"]+).*version *= *"((\d+)\.(\d+)[^"]*).*)/.exec(line);
```

Comment:
I think you can just check whether r is null which means that regex was not met, then exit 1
try/catch here seems overwhelming